Poker.AT/AT.Bridage/Real.cs

```diff
@@ -3,13 +3,19 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using AT.Domain;
 
 namespace AT.Bridge
 {
     class Real : TestsBridge
     {
-        //Here goes the Adapter implementation - for later use!
-        public int testCase1(int someParam)
+		public IList<Card> ShuffleCards(Deck TestDeck)
+		{
+			throw new NotImplementedException();
```

Comment:
for BE pull requests please don't leave: 
```c#
throw new NotImplementedException()
```
because the test will fail with the CI tool. 😃